app/controllers/public_health_controller.rb

```diff
@@ -92,8 +92,11 @@ def linelist(patients, workflow, tab)
                                'patients.date_of_birth, patients.assigned_user, patients.exposure_risk_assessment, patients.monitoring_plan, '\
                                'patients.public_health_action, patients.monitoring_reason, patients.closed_at, patients.last_date_of_exposure, '\
                                'patients.created_at, patients.updated_at, patients.latest_assessment_at, patients.latest_transfer_at, '\
-                               'patients.continuous_exposure, patients.head_of_household, jurisdictions.name AS jurisdiction_name, '\
-                               'jurisdictions.path AS jurisdiction_path')
+                               'patients.continuous_exposure, patients.head_of_household, patients.purged, patients.monitoring, patients.isolation, '\
+                               'patients.responder_id, patients.pause_notifications, patients.preferred_contact_method, '\
+                               'patients.last_assessment_reminder_sent, patients.preferred_contact_time, patients.extended_isolation, '\
+                               'patients.latest_fever_or_fever_reducer_at, patients.latest_positive_lab_at, patients.negative_lab_count, '\
+                               'jurisdictions.name AS jurisdiction_name, jurisdictions.path AS jurisdiction_path, jurisdictions.id AS jurisdiction_id')
```

Comment:
Why were these fields added?

Reply:
Since we are using select statement the objects within the AR Collection returned by the query did not have these attributes which are required by the methods that come after this.